Second round of edits

steffilazerte opened the pull request:
Starting on the second round of modifications to ARUtools

Including... 
- [x] Bug fixes
    - [x] #18 
- [X] General clean-up
    - [x] `fun_aru_samp()` (and family) [`sim_selection_weights()`, `calc_selection_weights()`, `sample_recordings()`] #23 
    - [x] `soxSpectrogram()` #12 
    - [x] `format_clip_wave()` family #8
    - [x] `acoustic_indices()` (formerly `getvals()`) #11
    - [x] `create_dirs()` (formerly `create_directory_structure()`) #13
    - [x] Use NSE for column names as appropriate #15 
    - [x] Standardize sorting output #19 
- [X] New functions
    - [X] Read log files from acoustic recording units (BarLT, ~~Song Meter 4, Song Meter 3, and Song Meter-Mini~~) #28
- [x] Tests
- [x] Documentation
    - [x] Function reference
    - [X] New example vignette for sampling
- ~~Add ability to pull timezone from BAR ARU sound file names~~ 
    - Leave for later #27

steffilazerte commented:
Hi @dhope, everything should be ready now, with the exception of reading log files and extracting the timezones. When you get a chance, could you peruse the functionality and close the issues you think are done (i.e., if you're happy with a function family, close the issue where we discussed things).

Anything that isn't quite as you like, please leave a comment and I'll tweak it accordingly!

I'll probably move to BASSr for the next little bit and will come back to the outstanding two things at the end.

dhope reviewed (commented):
Had a read through current state and didn't see any issues jump out. I'll leave this as a comment for now in case you want to make any more changes before closing the PR.

steffilazerte commented:
Hi @dhope, 
I think we're ready here too! 🎉 

Unfortunately I didn't have any leftover hours to address #27. 

However, 
- all the required functions are working well
    - #8 
    - #11 
    - #12 
    - #13 
    - #23)
- fixed bugs and added enhancements
     - #15
     - #18
     - #19
- added a new `clean_logs()` function #28 
- all functions have tests, examples and updated documentation
- I updated the coding to avoid warnings and notes for CRAN checks for these functions

So take a look and let me know how it goes, as always I'm happy to fix any problems you might find going forward.
